### PR TITLE
General: Fix Open File action to use main file of representation

### LINKED ISSUE #3760: Loader: Published versions menu entries errors and mismatches
**Describe the bug**
Published reviews from local or farm can't be played directly in loader (both in app and standalone).
This happens only if the review is published using either "local" or "On farm" (tested on Nuke)
After publishing Loader displays a "Play <repre>" entry for all the representations in addition to the usual "Open in DJV"
If any entry other than "Open in DJV" and "Play (exr)" is selected loader returns a list index out of range error
If version is published using "Use existing frames" the play options do not appear at all.
Also, the menu entries are being constructed based on last publish, so if someone publishes on farm loader gets the options, afterwards if someone publishes a new version with use existing frames the previous play entries disappear.

**To Reproduce**
1. Publish from nuke with either "local" or "On farm" and Review checked
2. open loader (either from tray or in app)
3. select the published asset and right click
4. See error when selecting all play entries other than "Play (exr)"
5. republish a new version with "Use existing frames"
6. open loader, select asset and right click
7. Play entries disappear, only "Open in DJV (exr)" is available

**Expected behavior**
same behaviour for all three publish modes ("Use existing frames, "local", "On farm").
Best if we get the play entries, to see a mov i need to copy file to desktop every time or use the delivery option and navigate there.

**Screenshots**
Version published with either "Local" or "On Farm" 
repre

Error when selecting any play entry aside from exr
error

Play entries disappears when republishing a new version with "Use Existing Frames"
Use-External-Frames

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Host: Tested only in Nuke 12.1, 13.1 and 13.2

[cuID:[OP-3853](https://app.clickup.com/t/2t3wjy8)]

### PR DESCRIPTION
**Pull request by BigRoy**
## Brief description

This should fix #3760 or at least a part of it. This should fix "Open file" not being able to open some of the representations.

## Description

For image sequences this should open the same first file as before.

## Testing notes:

1. Test the **Open** action

### CODE REVIEW

**Comment by BigRoy** [comment]
**Note: This does not fix the full #3760 issue but only a part of it.** This PR is currently set to close/resolve that Issue when getting merged and I'm unable to remove that linked issue. So probably best to hold off merging until that issue is also fixed (or merge this and quickly re-open the issue).

**Review by iLLiCiTiT — APPROVED** [review]
Good job

**Pull request merged**
